contrib/testing/kind-common.yaml

```diff
@@ -4,6 +4,10 @@ operator:
   nodeSelector:
     kubernetes.io/os: linux
     kubernetes.io/hostname: kind-worker
+  image:
+    pullPolicy: IfNotPresent
```

Comment:
Isn't `Always` a better option in dev workflow? Typically, developers would iterate over changes, and expect images to be re-loaded.

Reply:
Normally yes, but the `-fast` workflow does a `docker cp` of the generated binaries, so there's no point in re-pulling the image. It won't have the desired changes anyways.

Reply:
Why not make this part of `contrib/testing/kind-fast.yaml` instead? This way we won't be changing the behavior of "non-fast" dev workflows

Reply:
:facepalm: yes, of course. `make kind` already sets `pullPolicy: Never`.. which I added.

Reply:
Done.